src/components/Navbar/Navbar.tsx

```diff
@@ -27,12 +27,12 @@ const Navbar = () => {
       <nav className={classNames(styles.container, { [styles.hiddenNav]: !showNavbar })}>
         <Banner
           shouldShowPrefixIcon={false}
-          text={t('beyond-ramadan')}
+          text={t('best-days-banner')}
           ctaButton={
             <Button
-              href={getBeyondRamadanNavigationUrl()}
+              href={getDonationUrl()}
```

Comment:
Please take a look at [this](https://github.com/quran/quran.com-frontend-next/pull/2167/files#diff-3fb9bb17462f833c508f7e5c5d2d9273592b779c587257b5ea04cb1577d32091) change